tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -108,23 +108,23 @@ steps:
                   exit 1
                 fi
             else
-              feedId=$(grep -oP '_packaging/\K[^/]+(?=/npm/)' publish_log)
+              feedName=$(grep -oP '_packaging/\K[^/]+(?=/npm/)' publish_log)
 
```

Comment:
I liked the intermediate state of printing `feedName` for troubleshooting purposes, since we have no other way to know what it "resolved" to; we can't look at the publish_log after the pipeline runs. `$(release)` we do print in the Parameter step, IIRC. I'd also add an echo for `parameters.artifactPath`, because we've already done a few strides towards completely removing the distinction between them for the purposes of publishing packages in pipelines; at some point I'd like to clean it up from here, so knowing what was passed could be handy.